internal/symbolizer/symbolizer.go

```diff
@@ -282,9 +283,19 @@ func newMapping(prof *profile.Profile, obj plugin.ObjTool, ui plugin.UI, force b
 			continue
 		}
 
+		if m.File == "" {
```

Comment:
We will diagnose missing filenames when we actually try to open them below (line 302).
Previously if the filename was empty we would silently skip it.